cmd/aggregated-apiserver/app/options/options.go

```diff
@@ -84,7 +84,7 @@ func (o *Options) Run(ctx context.Context) error {
 
 	restConfig, err := clientcmd.BuildConfigFromFlags(o.Master, o.karmadaConfig)
 	if err != nil {
-		return fmt.Errorf("error building kubeconfig: %s", err.Error())
+		return fmt.Errorf("error building kubeconfig: %w", err)
```

Comment:
I'm ok with this(wrap the error). But I want to clarify that it's not necessary to do that as we won't check the error types.